README.md

```diff
@@ -76,4 +72,4 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Fellow Micronauts
+- Levina Nadezhda ([@petushka1](https://github.com/petushka1)) ~ Co-Partner on the original design
```

Comment:
- [ ] Please add a `License` section with the `MIT` license at the end of your README file to make it more professional.

Reply:
I understand about **Licence** but what I don't understand is what **MIT** has to do with this project.
Kindly elaborate.

Reply:
- Resolved 👍🏻 smile